### PR TITLE
Better debug messages

### LINKED ISSUE #11: Add better debugging messages
When the manifest validation failed, it does not report which line and column has problem, suggest to make changes to function logValidationReport() to add the printing of 'line', 'column' and 'code' to explicitly state where and what the problems are.    This would have saved tons of time in debugging the manifest file.

### PR DESCRIPTION
**Pull request by MIchaelMainer**
Closes #11

Before
image

Now
image

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -124,18 +124,62 @@ function logValidationReport(obj, name) {
...
+    }
+    ++n;
+  }
+}
+
+// Provide detailed warning information provided in the validationReport.warnings object.
+function logWarnings(warnings) {  
+  let n = 1;
+  for (let w of warnings) {
+    console.log(`  ${chalk.bold.red('Warning  #' + n + ': ')}`);
```
Comment (cbales) [inline src/index.ts]: Should this be yellow?

File: src/index.ts [hunk 2]
```diff
@@ -179,7 +160,22 @@ function logWarnings(warnings) {
...
+
+// Logs out the Code, Column, and/or Line value returned from the validation service.
+// param: obj - either an validationReport.errors[x], validationReport.warnings[x], or validationReport.infos[x] object.
+function logDetails(obj) {
+  console.log('  - ' + obj.title + ': ' + obj.detail + ' (link: ' + obj.link + ')');
+  if (obj.code) {
+    console.log('  - Code: ' + obj.code);
+  }
+  if (obj.column) {
+    console.log('  - Column: ' + obj.column);
```
Comment (cbales) [inline src/index.ts]: Switch column and line reporting, or put in () as (line, column)

File: src/index.ts (hunk 1 above)
Comment (MIchaelMainer) [inline src/index.ts]: Yes! Thank you!

File: src/index.ts (hunk 2 above)
Comment (MIchaelMainer) [inline src/index.ts]: I've switched line and column. Thank you.

**Comment by MIchaelMainer** [comment]
I've changed the output field from Code to Details.

**Comment by MIchaelMainer** [comment]
image

**Pull request merged**